config/config.py.template

```diff
@@ -143,7 +143,7 @@ COMMON_JAVA_OPTS = [
     # Fraction of JVM memory used for caching RDDs.
     JavaOptionSet("spark.storage.memoryFraction", [0.66]),
     JavaOptionSet("spark.serializer", ["org.apache.spark.serializer.JavaSerializer"]),
-    JavaOptionSet("spark.executor.memory", ["9g"]),
+    # JavaOptionSet("spark.executor.memory", ["9g"]),
```

Comment:
Per @JoshRosen, if we leave this out the default setting in Spark will let the executor grab as much memory as possible, which is what we want.

Did I get that correct, Josh?